assets/js/modules/adsense/components/common/AdBlockingRecoveryCTA.test.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * AdBlockingRecoveryCTA component tests.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import AdBlockingRecoveryCTA from './AdBlockingRecoveryCTA';
+import { render, provideModules } from '../../../../../../tests/js/test-utils';
+import {
+	AD_BLOCKING_RECOVERY_SETUP_STATUS_SETUP_CONFIRMED,
+	MODULES_ADSENSE,
+} from '../../datastore/constants';
+import {
+	ACCOUNT_STATUS_PENDING,
+	ACCOUNT_STATUS_READY,
+	SITE_STATUS_ADDED,
+	SITE_STATUS_READY,
+} from '../../util';
+
+describe( 'AdBlockingRecoveryCTA', () => {
+	it.each( [
+		[
+			'should not render the CTA when Adsense account status is not ready',
+			ACCOUNT_STATUS_PENDING,
+			SITE_STATUS_READY,
+			'',
+		],
+		[
+			'should not render the CTA when Adsense site status is not ready',
+			ACCOUNT_STATUS_READY,
+			SITE_STATUS_ADDED,
+			'',
+		],
+		[
+			'should not render the CTA when Ad blocking recovery status is not an empty string',
+			ACCOUNT_STATUS_READY,
+			SITE_STATUS_ADDED,
+			AD_BLOCKING_RECOVERY_SETUP_STATUS_SETUP_CONFIRMED,
+		],
+	] )(
+		'%s',
+		( { accountStatus, siteStatus, adBlockingRecoverySetupStatus } ) => {
```

Comment:
This is not correct, rows from the table are destructed as arguments for the test function. So the correct signature should be as following:
```suggestion
		( testName, accountStatus, siteStatus, adBlockingRecoverySetupStatus ) => {
```